packages/installations/src/util/errors.ts

```diff
@@ -30,6 +32,14 @@ export const enum ErrorCode {
 const ERROR_DESCRIPTION_MAP: { readonly [key in ErrorCode]: string } = {
   [ErrorCode.MISSING_APP_CONFIG_VALUES]:
     'Missing App configuration value: "{$valueName}"',
+  [ErrorCode.INVALID_APP_ID_FORMAT]:
+    'Please set your Application ID. A valid Firebase App ID is required to communicate ' +
```

Comment:
The message sounds like the app id is missing, but the error is actually a malformed appId. I would also just use `appId` which matches the name in the config object.

Reply:
makes sense. done